website/examples.js

```diff
@@ -0,0 +1,114 @@
+document.addEventListener('DOMContentLoaded', () => {
+    console.log('Docs loaded');
+
+    var startersCard = document.getElementById('starters-card');
+    var ttlCard = document.getElementById('ttl-card');
```

Comment:
These functions return a single element and not a list like the others due to the HTML expectation of a single ID.